dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/channel/internal/HttpRequestMessageImpl.java

```diff
@@ -2209,6 +2209,6 @@ private boolean isPrivateHeaderTrusted(HeaderKeys key) {
 
     @Override
     public long getEndTime() {
-        return 0;
+        this.getServiceContext().getResponseStartTime();
```

Comment:
Can we add a comment here as to why the request endtime is the response start time?